### PR TITLE
Fix interpretations in mitotip_score_fixed_del.csv

### PR DESCRIPTION
**Pull request by anjalijain22**
- Addresses issue #14 
- Changes have been made to two files:
1. **mitylib/annot/mitotip_score_fixed_del.csv:** The MitoTip_interpretations column values have been changed. The values can be `likely pathogenic`, `possibly pathogenic`, `likely benign` or `possibly benign` depending on the Mitotip_Percentile column value according to [this table](https://www.mitomap.org/MITOMAP/MitoTipInfo). Additionally, some variants have values "Confirmed pathogenic" according to the [list of confirmed pathogenic variants on MitoMap](https://mitomap.org/foswiki/bin/view/MITOMAP/MutationsRNACfrm).
2. **mitylib/report.py:** Updated documentation for "MitoTip_interpretation" to include Confirmed pathogenic variants.

- Known conflict:
1. Some variants having `Confirmed pathogenic` status in the MitoTip_interpretation column in mitylib/annot/mitotip_score_fixed_del.csv may have a `reported` value in the "status_mitomap" column of mitylib/annot/mitomap_panel_annotations.csv. 
2. Next step: Update mitylib/annot/mitomap_panel_annotations.csv file to the latest version

### CODE REVIEW

File: mitylib/report.py [hunk 1]
```diff
@@ -831,7 +831,11 @@ def do_report(vcf, prefix=None, min_vaf=0.0, out_folder_path = "."):
         "MitoTip_interpretation: Variants in the upper 25th percentile are "
         "deemed likely pathogenic, 50-75th percentile as possibly "
         "pathogenic; 25-50th percentile as possibly benign; "
-        "bottom 25th percentile as likely benign. see "
+        "bottom 25th percentile as likely benign. "
+        "Variants reported as Confirmed pathogenic have additional lines "
```
Comment (drmjc) [inline mitylib/report.py:835]: Confirmed Pathogenic in MITOMAP

**Review by drmjc — CHANGES_REQUESTED** [review]
thanks so much for the PR.
Can you please tweak the wording in report.py as noted, and version bump 'mitylib/_version.py' to 0.3.1

File: mitylib/report.py (hunk 1 above)
Comment (anjalijain22) [reply mitylib/report.py]: Done!

**Review by drmjc — APPROVED** [review]

**Comment by drmjc** [comment]
Thanks @anjalijain22.

@joecop can you please give me admin access again (and for clinsv)? thanks

**Pull request merged**